src/styles/_catalog.scss

```diff
@@ -14,6 +26,10 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    @media (min-width: 488px) {
+      margin-bottom: 0;
+    }
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
You should delete margin-right as content isn't centered because of it
![image](https://user-images.githubusercontent.com/102293862/187474043-1530af0b-fa4a-4ce6-9d19-3d4608b3313a.png)